pkg/deployment/members.community.go

```diff
@@ -18,6 +18,7 @@
 // Copyright holder is ArangoDB GmbH, Cologne, Germany
 //
 
+//go:build !enterprise
```

Comment:
it comes from go 1.7 - are you sure we want to add it right now

Reply:
removed